platform/platform_interface.go

```diff
@@ -47,7 +47,9 @@ type Platform interface {
 	MigratePersistentDisk(fromMountPoint, toMountPoint string) (err error)
 	GetEphemeralDiskPath(diskSettings boshsettings.DiskSettings) string
 	IsMountPoint(path string) (result bool, err error)
+	FindDeviceMatchingMountPoint(string) (string, bool, error)
```

Comment:
talking with @tylerschultz about this interface, let's try to keep platform interface minimal. we should probably have IsMountPoint return (string, bool, error). wdyt? also name is not necessarily related to devices, since other things could be mounted.